backend/utils/llm

```diff
@@ -0,0 +1,13 @@
+import os
+from dotenv import load_dotenv
```

Comment:
Depending on who merges first this will need adding to the `requirements.txt` file I added. I've added `mistralai` already just not `dotenv`